q/util/util.q

```diff
@@ -13,3 +13,75 @@
 // @return A byte vector containing the contents of the file.
 // @see read1
 .finos.util.read1f:{r:{y,read1 x}[h:hopen`$":fifo://",1_string x]over();hclose h;r}
+
+.finos.util.compose:('[;])/
+
+// create a list. e.g. list(`a;1) -> (`a;1)
+.finos.util.list:{$[104h=type x;1_-1_get x;x]}
```

Comment:
This needs more examples for why it is useful (also considering we already have a function to create a list, it's called `enlist`). For most types it returns the parameter unchanged, which means if it's an atom the result is not a list, contrary to what the description suggests. The only special case is for projections, for which we drop the function that is being projected, so we only keep the bound arguments... but then we also drop the last of those arguments?

Reply:
the point is to let you write in "trailing delimiter" style,

```
foo:(
    1;
    2;
    )
```

which has advantages for source control and general future-proofing

i can add an example to that effect

it's also used by `util.dict` and `util.table` to provide the same feature to them